### DISCUSSION TITLE
Feature: refocus the interface on the content (Github mode)

### DISCUSSION THREAD

**Discussion by le-u-f** [Thinkmill/keystatic#759] [Ideas]
Hi,

Would it be possible to find a way to simplify the interface (in Github mode) to avoid anything related to branch management in order to refocus the interface on the content.

Suggestions:
- Hide the branch management interface?
- Add a "content" branch feature?
- Something else?

Related (closed) PR:
https://github.com/Thinkmill/keystatic/pull/751

**Comment by inwardmovement** [comment]
Indeed, we should be able to hide any dev related UI or concepts for editors, and just let them "edit the site".

No prompting for creating branches, etc. If the editing/content branch is protected, then it cannot be edited and that's it. Do not protect it if edition must be allowed.

Imagine people coming from Wordpress and you have to explain them the branching stuff...
